Создан фрагмент для списка. Создан фрагмент на основе активи, отображ…

demqn83 opened the pull request:
![movie_list](https://user-images.githubusercontent.com/73488383/100385452-b6024700-3033-11eb-8989-856f110d9f35.jpg)
…ающий информацию о фильме. Добавлен обработчик клика для открытия фрагмента.

demqn83 committed:
Создан фрагмент для списка. Создан фрагмент на основе активи, отображающий информацию о фильме. Добавлен обработчик клика для открытия фрагмента.

ivanlardis commented:
![image](https://user-images.githubusercontent.com/11761610/100478728-29bc5680-30fd-11eb-8627-acde8216c78b.png)
В альбомной ориентации не помещается на экран и не скролится

ivanlardis commented:
![image](https://user-images.githubusercontent.com/11761610/100478816-6ee08880-30fd-11eb-85d6-01a574b33504.png)
Кнопка назад не работает
И находится очень близко с текту back

ivanlardis commented on app/src/main/java/ru/demqn/appname/FragmentMoviesList.kt:
```diff
@@ -0,0 +1,61 @@
+        listener = null
+    }
+
+    fun setClickListener(l: TransactionsFragmentClicks?) {
```
Тут не нужен этот метод
Обьект активити призодит тут
 override fun onAttach(context: Context) {
        super.onAttach(context)
        if (context is TransactionsFragmentClicks) listener = context
    }
Когда фрагмент атачится к активити
Там же у тебя уже проставляется листенер

ivanlardis commented on app/src/main/java/ru/demqn/appname/MainActivity.kt:
```diff
@@ -2,10 +2,34 @@ package ru.demqn.appname
+class MainActivity : AppCompatActivity(), FragmentMoviesList.TransactionsFragmentClicks {
+
+    private val rootFragment =
+        FragmentMoviesList().apply { setClickListener(this@MainActivity) }
```
.apply { setClickListener(this@MainActivity) }
Тут не нужно, слудует из-за коментария выше

ivanlardis commented on app/src/main/res/layout/fragment_movies_list.xml:
```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
скрол всетаки нужен
Для альбомной ориетации

ivanlardis commented on app/src/main/res/layout/fragment_movies_list.xml:
```diff
@@ -0,0 +1,163 @@
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="20dp"
+        android:layout_marginTop="378dp"
```
Давай начнем привязыватся не к верху экрана
А к тексту который находится выше этого

ivanlardis commented on app/src/main/res/layout/fragment_movies_list.xml:
```diff
@@ -0,0 +1,163 @@
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <ImageView
+        android:id="@+id/ic_like_image_view"
```
Будет круто если при нажатии на like
Он будет менять цвет
На красный и назад

ivanlardis commented on app/src/main/java/ru/demqn/appname/MainActivity.kt:
```diff
@@ -2,10 +2,34 @@ package ru.demqn.appname
+                add(R.id.container_view, rootFragment)
+                commit()
+            }
+        }
```
![image](https://user-images.githubusercontent.com/11761610/100484005-cb4aa480-310b-11eb-941c-a5e15ec9fc81.png)
Правильнее взять реализацию из лекции
В воркшопе было не правильно

demqn83 committed:
Для альбомной ориентации добавлен скролл.
Реализовано поведение назад на надписи "Back".
Изменены привязки у вью.
Изменена реализация вызова фрагмента.
